{"frames":[{"func":"data.frame(y = sample(population, 40, replace = FALSE))","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"sample(population, 40, replace = FALSE)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"x[sample.int(length(x), size, replace, prob)]","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"`[.tbl_df`(x, sample.int(length(x), size, replace, prob))","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"sample.int(length(x), size, replace, prob)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in sample.int(length(x), size, replace, prob) : \n  cannot take a sample larger than the population when 'replace = FALSE'\n"}